war/src/test/java/io/lumeer/remote/rest/UserServiceIntegrationTest.java

```diff
@@ -0,0 +1,171 @@
+package io.lumeer.remote.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.lumeer.api.dto.JsonOrganization;
+import io.lumeer.api.dto.JsonPermission;
+import io.lumeer.api.dto.JsonPermissions;
+import io.lumeer.api.model.Role;
+import io.lumeer.api.model.User;
+import io.lumeer.core.AuthenticatedUser;
+import io.lumeer.storage.api.dao.OrganizationDao;
+import io.lumeer.storage.api.dao.UserDao;
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import javax.inject.Inject;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@RunWith(Arquillian.class)
+public class UserServiceIntegrationTest extends ServiceIntegrationTestBase {
+
+   private static final String USER = AuthenticatedUser.DEFAULT_EMAIL;
+
+   private static final String USER1 = "user1@gmail.com";
+   private static final String USER2 = "user2@gmail.com";
+   private static final String USER3 = "user3@gmail.com";
+
+   private static final Set<String> GROUPS;
+
+   static {
+      GROUPS = new HashSet<>(Arrays.asList("group1", "group2", "group3"));
```

Comment:
Why is this in a `static` block?

(Was there something wrong with `private static final Set<String> GROUPS = new HashSet<>(Arrays.asList("group1", "group2", "group3"));`?)

Reply:
I copied it from another test, but you are right there's no reason to do it in static block in this situation.